tests/topotests/lib/topogen.py

```diff
@@ -492,7 +492,19 @@ def stop_topology(self):
                 "Errors found post shutdown - details follow: {}".format(errors)
             )
 
-        self.net.stop()
+        try:
+            self.net.stop()
+
+        except OSError as error:
+            # OSError exception is raised when mininet tries to stop switch
+            # though switch is stopped once but mininet tries to stop same
+            # switch again, where it ended up with exception
+
+            logger.info(error)
+            logger.info(
+                "Exception is raised when mininet tries to stop switch"
+                "though switch is stopped once but mininet tries to stop same switch again."
```

Comment:
this could be simpler - how about: "Exception ignored: switch is already stopped"

Reply:
> this could be simpler - how about: "Exception ignored: switch is already stopped"

Hi Mark Stapp,

Modified the title of the PR.

Thanks & Regards,
Bharath